demo/app/components/drawer/drawer.component.html

```diff
@@ -41,6 +44,12 @@
           </ts-radio-group>
         </p>
       </div>
+      <div style="min-height: 600px; ">
```

Comment:
It would be amazing to have 'real' content in here for a realistic demo. Maybe just some duplicated paragraphs with spacing?

eg

```html
  <p tsVerticalSpacing>
    Doloribus quis exercitationem sint sequi illo. Assumenda ducimus quae quam. Ipsum ut dolor.
  </p>
```

Bonus points if the viewer can check/uncheck a checkbox to show/hide the content so they can see with/without scrollable content. (not necessary though 😄)